src/java.base/share/classes/java/security/spec/PSSParameterSpec.java

```diff
@@ -107,15 +107,15 @@
 
     /**
      * The PSS parameter set with all default values
-     * @deprecated The default values defined in the ASN.1 encoding in
-     *         PKCS#1 may become obsolete as time progresses. Thus, it is
-     *         recommended to explicitly specify all desired parameter
-     *         values with
+     * @deprecated This field uses the default values defined in the ASN.1
+     *         encoding in PKCS#1 which may become obsolete as time progresses.
+     *         Thus, it is recommended to create a new PSSParameterSpec object
```

Comment:
Should `PSSParameterSpec` be inside a `{@code ...}`?

Reply:
Yes, good catch. Will fix. Thanks!